src/components/actionPayloadRenderers/SessionPayloadRenderer.tsx

```diff
@@ -25,11 +25,14 @@ export default class Component extends React.Component<Props, {}> {
             ? null
             : sessionAction.renderValue(Util.createEntityMapFromMemories(entities, memories), { fallbackToOriginal: true })
 
-        return <div className={`${OF.FontClassNames.mediumPlus} cl-text-payload cl-font--emphasis`}>
-            <div className="cl-text-payload__string" data-testid="action-scorer-text-response">
-                EndSession: {(renderStringUsingCurrentMemory === null)
-                    ? `${renderStringUsingEntityNames}`
-                    : `${renderStringUsingCurrentMemory}`
+        return <div className={`${OF.FontClassNames.mediumPlus}`}>
+            <div className="cl-payloadrenderer-session-primary" data-testid="action-scorer-session-response">
```

Comment:
In my opinion the word `extended` is too generic. It doesn't explain what the extension is or why it's extended. None our other renderers have this concept of extended either so there is no understood meaning. I would prefer something like `-system` and `-user` since `EndSession` is hard coded and won't change and the other field is optional and from the user.

That all being said it's not that important. Only a test id, doesn't effect product function, and isn't visible to customer so I'd still approve.

Reply:
Good suggestions. I'm going to switch to -user as that capture's the intent of the code block